### PR TITLE
resolves #189 Use other delimiter for stylesheet override 

### LINKED ISSUE #189: Use other delimiter for stylesheet override
I would suggest to use a different delimiter than `;` for accepting multiple stylesheets, e.g.  `,`
Because `;` is also a command delimiter for shell so it can easily lead to `command not found` errors, which is not obvious and can take a while to understand the reason. e.g.
```
$ asciidoctor-pdf -a stylesheet=asciidoctor.css;document.css mydoc.adoc
...
bash: document.css: command not found
```
At least adjust the example in the readme:
> The stylesheet attribute can accept multiple semi-colon delimited values (without spaces). For example -a stylesheet=asciidoctor.css;override.css. This could be used to begin with a base stylesheet and then apply supplementary content.

### PR DESCRIPTION
**Pull request by makon**
This resolves #189 by changing the stylesheet delimiter to be a comma `,`.
Be aware that this is a breaking change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,16 +201,25 @@ The title page is enabled if either of these conditions are met:
 $ asciidoctor-pdf document.adoc -a title-page
 ```
 
-**Additional styles**
+**Custom styles**
 
-You can provide a custom stylesheet using the `stylesheet` attribute.
-You can also specify where the stylesheet is located with the `stylesdir` attribute.
+You can provide a custom stylesheet using the `stylesheet` attribute. A custom stylesheet does completely replace the default stylesheets.
```
Comment (Mogztter) [inline README.md]: Should we say the "default stylesheet" (without "s")?

File: README.md [hunk 2]
```diff
@@ -201,16 +201,25 @@ The title page is enabled if either of these conditions are met:
...
 
-The `stylesheet` attribute can accept multiple semi-colon delimited values (without spaces).
-For example `-a stylesheet=asciidoctor.css;override.css`.
-This could be used to begin with a base stylesheet and then apply supplementary content.
+The `stylesheet` attribute can accept multiple comma delimited values (without spaces).
+This can be used to begin with a base stylesheet and then apply supplementary content.
+
+    $ asciidoctor-pdf document.adoc -a stylesheet=custom.css,override.css
+
+It's also possible to use the default stylesheets and add custom styles with a custom stylesheet. All default stylesheets are available under the prefix `asciidoctor-pdf/css/`:
```
Comment (Mogztter) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -201,16 +201,25 @@ The title page is enabled if either of these conditions are met:
...
 
     $ asciidoctor-pdf document.adoc -a stylesheet=custom.css
 
-The `stylesheet` attribute can accept multiple semi-colon delimited values (without spaces).
-For example `-a stylesheet=asciidoctor.css;override.css`.
-This could be used to begin with a base stylesheet and then apply supplementary content.
+The `stylesheet` attribute can accept multiple comma delimited values (without spaces).
+This can be used to begin with a base stylesheet and then apply supplementary content.
+
+    $ asciidoctor-pdf document.adoc -a stylesheet=custom.css,override.css
```
Comment (Mogztter) [inline README.md]: I wonder if we should use the "safe" syntax with double quotes (even if in this example it's not required)... what do you think?

Comment (makon) [reply README.md]: Good point. I adjusted it

File: README.md (hunk 1 above)
Comment (makon) [reply README.md]: Done

**Pull request merged**